Source/WebCore/platform/mediastream/mac/ScreenCaptureKitCaptureSource.mm

```diff
@@ -214,51 +214,44 @@ - (void)outputVideoEffectDidStopForStream:(SCStream *)stream
         return;
     }
 
+    m_isPrewarming = true;
+    m_whenReadyCallback = WTFMove(callback);
```

Comment:
Nit: moving this line before the `if (m_isRunning)` check might make this slightly clearer to someone who doesn't know this code